### ISSUE TITLE
Messages sent by inline bots are not announced to be sent via inline bots when arrowing

### ISSUE THREAD

**Issue by sukiletxe** [UnigramDev/Unigram#2236]
This is similar to #2200, and the steps to verify it are the same.
